docs/manage/_content.rst

```diff
@@ -160,29 +197,28 @@ If you want to make available the content you have imported on your Kolibri devi
 #. Wait for Kolibri to export the selected content and click **Close**.
 #. Once the export is finished, safely disconnect the drive according to the recommended procedure for your operating system, and proceed to import channels on other devices.
 
-.. note:: This procedure makes a copy of the ``content`` folder located inside the ``.kolibri`` folder on your hard disk, and places it in the ``KOLIBRI_DATA`` folder on the selected local drive. This structure is recognized by the **Import from local drive** command.
+    .. note:: This procedure makes a copy of the ``content`` folder located inside the ``.kolibri`` folder on your hard disk, and places it in the ``KOLIBRI_DATA`` folder on the selected local drive. This structure is recognized by the **Import from local drive** command.
 
-	.. image:: img/kolibri-data-osx.png
-	  :alt: structure of the local drive folders with exported content channels
+        .. figure:: img/kolibri-data-osx.png
+          :alt: structure of the local drive folders with exported content channels
 
 Delete Channel
 --------------
 
 To delete a content channel from your device, follow these steps.
 
-#. Click **Actions** for the channel you want to delete.
+#. Click **Options** for the channel you want to delete.
 #. Select **Delete** option.
-#. Click **Delete** to proceed, or **Cancel** to exit without deleting the channel.
 
-  .. image:: img/delete-channel.png
-    :alt: delete content channel from your device
+    .. figure:: img/delete-channel.png
+      :alt: 
+    
+#. Click **Delete** to proceed, or **Cancel** to exit without deleting the channel.
 
 
 
-Peer-to-Peer Content Synchronization
-------------------------------------
 
-``Stay tuned!``
 
+.. Peer-to-Peer Content Synchronization
 
-.. Not yet. Peer to peer sync is a priority for Kolibri in the near future, but is not available yet.
+.. Hiding this as it is covered under import from local network, right?
```

Comment:
@indirectlylit Is this a correct supposition?

Reply:
> If yes, maybe I should delete it completely?

yes, agreed